src/framework/MOM_diag_mediator.F90

```diff
@@ -122,6 +142,25 @@ module MOM_diag_mediator
   type(diag_grids_type), dimension(:), allocatable :: diag_grids      !< Primarily empty, except h field
 end type diag_grid_storage
 
+!> integers to encode the total cell methods
```

Comment:
Yes, for 2d fields can use the same 3 digit code, although the v-code (3rd digit from left) might not be needed. 
All 2d diagnostics in diag_table.MOM6 come with a "Point" for the v_cell_method, so I assume they are all either area averaged (MMP) or summed or sampled, in other words, no 2d field is volume averaged! Is that correct?
Although I coded the "MMM" for 2d fields I just found out it's never used and am going to delete that case.

Reply:
Could I suggest that we do the enumeration using 1,2,3 instead of 0,1,2, so that the 2-d variant PS (which would currently resolve to 1) can not be confused with the 3-d PPS (which would also resolve to 1).  With 1,2,3, PS becomes 12, whereas PPS becomes 112.  0 could be reserved for no-axis.